src/frontend/components/StudentLiveWrapper/StudentLiveLayout/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Box } from 'grommet';
+import { normalizeColor } from 'grommet/utils';
+import styled from 'styled-components';
+
+import { theme } from 'utils/theme/theme';
+
+const LiveVideoInformationBarWrapper = styled(Box)`
+  -webkit-box-shadow: 0px 0px 7px 5px ${normalizeColor('shadow-1', theme)};
+  box-shadow: 0px 0px 7px 5px ${normalizeColor('shadow-1', theme)};
+`;
+
+interface LiveStudentLayoutProps {
+  actionsElement: React.ReactElement;
+  isPanelOpen?: boolean;
+  liveTitleElement: React.ReactElement;
+  mainElement: React.ReactElement;
+  sideElement?: React.ReactElement;
+}
+
+export const StudentLiveLayout = ({
+  actionsElement,
+  isPanelOpen,
+  liveTitleElement,
+  mainElement,
+  sideElement,
+}: LiveStudentLayoutProps) => {
+  return (
+    <Box>
+      <Box direction="row">
+        {/* main view rendering player or conf to stream */}
+        <Box basis="full">{mainElement}</Box>
+
+        {sideElement && (
+          // for now panel is kept in the DOM but hiden to mount chat and therfore connect to XMPP
+          <Box data-testid="panel-id" basis="1/4" hidden={!isPanelOpen}>
```

Comment:
```suggestion
          <Box data-testid="panel-id" basis="1/4" hidden={isPanelHidden}>
```
Maybe rename and invert condition ?

Reply:
I'm not a big fan:
- the reverse condition will juste be in the parent component
- the goal is yet to unmount and remove from the dom this component when not visible, but to be able to use XMPP at any stage without opening the tab on startup I just hide it with CSS